pkg/translator/pprof/profiles_to_pprof.go

```diff
@@ -86,7 +86,7 @@ type lookupTables struct {
 	lastStackTableIdx int32
 }
 
-func convertPprofToPprofile(src *profile.Profile) (*pprofile.Profiles, error) {
+func ConvertPprofToPprofile(src *profile.Profile) (*pprofile.Profiles, error) {
```

Comment:
@florianl is there a reason we return a pointer to pprofile.Profiles here? Can we return by value instead?

Reply:
@atoulme is the intention that this follow the existing practices done with the other signals where the exported type is just a wrapper around the pointer to the heap?